Libraries/Utilities/createStrictShapeTypeChecker.js

```diff
@@ -47,9 +48,12 @@ function createStrictShapeTypeChecker(
     for (var key in allKeys) {
       var checker = shapeTypes[key];
       if (!checker) {
+        const stack =  parseErrorStack(new Error());
+        const renderFrame = stack.find((frame) => frame.methodName.endsWith('render'));
         invariant(
           false,
           `Invalid props.${propName} key \`${key}\` supplied to \`${componentName}\`.` +
+            `\nStack: ${JSON.stringify(renderFrame)}` +
```

Comment:
Do you mean here https://github.com/skv-headless/react-native/blob/warning-stacktrace/Libraries/JavaScriptAppEngine/Initialization/updateExceptionWithSourceMap.js ?

No it doesn't throw a redbox only yellowbox.